### PR TITLE
Port some InChI functions

### PR DESCRIPTION
**Pull request by maneetgoyal**
@qur786 Please take up this PR.

### CODE REVIEW

**Comment by maneetgoyal** [comment]
>@maneetgoyal sir I have tried different testing techniques for "freeStructFromInchi" but it's not working . Can you please look at it .

Taking a look

**Comment by maneetgoyal** [comment]
@qur786 Updated the tests; passing now.

**Comment by maneetgoyal** [comment]
Let me know once you've reviewed. Will merge thereafter.

File: tests/ffis.test.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 // @ts-nocheck There are some issues in the Definitely Typed packages of the "ref" related dependencies
-import { INCHIAPI, inchi_InputINCHI, inchi_Output } from "../src";
+import { INCHIAPI, inchi_InputINCHI, inchi_Output, inchi_OutputStruct, inchi_OutputStructEx } from "../src";
 import refNAPI from "ref-napi";
 import { readFileSync } from "fs";
 import { join } from "path";
 
+
+
```
Comment (maneetgoyal) [inline tests/ffis.test.ts:8]:
```suggestion
```

**Comment by qur786** [comment]
> Let me know once you've reviewed. Will merge thereafter.

sure sir

**Comment by maneetgoyal** [comment]
CI will fail so don't worry. Ref. https://github.com/manufac-analytics/inchi/issues/15

**Review by qur786 — APPROVED** [review]
all good sir 👍

**Pull request merged**